src/components/Icon/Icon.stories.tsx

```diff
@@ -1,7 +1,27 @@
 import React from "react"
 import { storiesOf } from "@storybook/react"
 import Icon from "./index"
+import * as Icons from "./Icons"
+
+const allIcons = Object.values(Icons)
 
 storiesOf("Icon", module)
   .add("Basic Icon", () => <Icon type="oAddCircle" />)
   .add("Icon with color", () => <Icon type="oAddCircle" fill="red" />)
+  .add("All icons", () => {
+    const allIconsDOM = allIcons.map((i, index) => (
+      <div key={index} style={{ margin: 20 }}>
```

Comment:
Can we also add an `alt` attribute on the div?

Reply:
Why so? Afaik, it's of no use on the div...but acts as placeholder for images when they are being loaded. Am I missing something?